apps/desktop-wallet/src/api/queries/tokenQueries.ts

```diff
@@ -128,23 +128,20 @@ export const nftDataQuery = ({ id, tokenUri, networkId, skip }: NFTQueryProps) =
               return Promise.reject()
             }
 
-            // NOTE: Couldn't get the proper content type with axios
-            const dataTypeRes = nftData.image ? (await fetch(nftData.image)).headers.get('content-type') || '' : ''
+            const dataTypeRes = nftData.image ? (await axios.get(nftData.image)).headers['content-type'] || '' : ''
 
             const dataTypeCategory = dataTypeRes.split('/')[0]
 
             const dataType = dataTypeCategory in NFTDataTypes ? (dataTypeCategory as NFTDataType) : 'other'
 
             return matchesNFTTokenUriMetaDataSchema(nftData)
               ? { id, dataType, ...nftData }
-              : nftData.name
```

Comment:
Since in line 127 we ensured that name exists, there's no reason to check it here as well.